File: javascript/gpt/p23.js
// p23: generate three random integers > 1000, print them and sums
function randAbove1000() {
  return Math.floor(Math.random() * 9000) + 1001;
}
const a = randAbove1000();
const b = randAbove1000();
const c = randAbove1000();
console.log(`a : ${a}`);
console.log(`b : ${b}`);
console.log(`c : ${c}`);
const sum = a + b + c;
console.log(`sum: ${sum}`);
// JavaScript Number is double; to emulate long long we can use BigInt
const sum2 = BigInt(a) + BigInt(b) + BigInt(c);
console.log(`sum2: ${sum2}`);

File: python/gpt/p23.py
# p23: generate three random integers > 1000, print them and their sums
import random

a = random.randint(1001, 9999)
b = random.randint(1001, 9999)
c = random.randint(1001, 9999)
print(f"a : {a}")
print(f"b : {b}")
print(f"c : {c}")
sum1 = a + b + c
print(f"sum: {sum1}")
# sum2 using Python int (unbounded) but label as long long equivalent
sum2 = int(a) + int(b) + int(c)
print(f"sum2: {sum2}")
